lib/ai_bot/commands/search_command.rb

```diff
@@ -208,9 +208,14 @@ def process(**search_args)
           row = {
             title: post.topic.title,
             url: Discourse.base_path + post.url,
+            username: post.user&.username,
             excerpt: post.excerpt,
             created: post.created_at,
             category: category_names,
+            likes: post.like_count,
+            topic_views: post.topic.views,
+            topic_likes: post.topic.like_count,
+            topic_replies: post.topic.posts_count - 1,
```

Comment:
There is a `reply_count` on `topic` already

Reply:
I think it has slightly diff semantics

Reply:
@SamSaffron wow yeah I looked it up and tested it, and it's the count of the number of posts replying to other posts in the topic :S And it's not changed if the post is deleted.